bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/findreplacelogicstatus/IStatus.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.ui.internal.findandreplace.findreplacelogicstatus;
+
+/**
+ * Interface for statuses that can occur while performing
+ * Find/Replace-operations.
+ */
+public interface IStatus {
+	public <T> T visit(IStatusVisitor<T> visitor);
```

Comment:
Please rename this to `accept` (or comparable), as the `visit` method is (semantically) part of the visitor and not the visited object and I think `accept` is standard terminology according to the GoF pattern.
Sorry that I didn't see and mention this naming "issue" before.

Reply:
https://github.com/eclipse-platform/eclipse.platform.ui/commit/555a45e5a86580594a0b35108879fa33b98de879